Core/include/Acts/Propagator/detail/GenericDenseEnvironmentExtension.hpp

```diff
@@ -171,9 +181,15 @@ struct GenericDenseEnvironmentExtension {
             typename navigator_t>
   bool finalize(propagator_state_t& state, const stepper_t& stepper,
                 const navigator_t& /*navigator*/, const double h) const {
+    // using because of autodiff
+    using std::hypot;
```

Comment:
Where was this before?

Reply:
I think I accidentally removed this in a previous PR because I did not know about the autodiff thing. I add it back in here to make it consistent